src/libs/actions/IOU.js

```diff
@@ -246,14 +246,32 @@ function buildPayPalPaymentUrl(amount, submitterPayPalMeAddress, currency) {
  * @param {String} params.currency
  * @param {String} [params.submitterPhoneNumber] - used for Venmo
  * @param {String} [params.submitterPayPalMeAddress]
+ * @param {String} [params.comment]
+ * @param {String} [params.requestorEmail]
  */
 function payIOUReport({
-    chatReportID, reportID, paymentMethodType, amount, currency, submitterPhoneNumber, submitterPayPalMeAddress,
+    chatReportID,
+    reportID,
+    paymentMethodType,
+    amount,
+    currency,
+    submitterPhoneNumber,
+    submitterPayPalMeAddress,
+    comment,
+    requestorEmail,
 }) {
     Onyx.merge(ONYXKEYS.IOU, {loading: true, error: false});
+    const randomArray = new Uint32Array(10);
```

Comment:
This and `crypto.getRandomValues` seem like something that can be refactored out of the IOU action.

Reply:
Some sort of helper for this might be better.

Reply:
@Julesssss Thank you for the review! Updated the code, abstracted the idempotency key to helper function.